tasks/agent.py

```diff
@@ -618,6 +618,8 @@ def get_omnibus_env(
     # Forward the DEPLOY_AGENT variable so that we can use a higher compression level for deployed artifacts
     if os.environ.get('DEPLOY_AGENT'):
         env['DEPLOY_AGENT'] = os.environ.get('DEPLOY_AGENT')
+    if 'PACKAGE_ARCH' in os.environ:
```

Comment:
what will occur if it's not defined? Is it supposed to be defined all the time?

Reply:
It's expected to be defined for all build jobs, but if it isn't then we'll just use the same number of threads for building & compressing the package